src/utils/calculation.js

```diff
@@ -55,13 +61,28 @@ const calculate = async (url, cql, patientId, periodStart, periodEnd) => {
 
   relevantResults.forEach((r) => {
     // Only grab the value that the definition returns
-    const value = r.resource.parameter.find((p) => p.name === 'value');
+    const { value, valueType } = getValueAndType(r.resource.parameter);
     const populationIdentifier = r.resource.id.toLowerCase().replace(/\s/g, '_');
 
-    // NOTE: using === 'true' to get a boolean value from the string that is returned from cqf-ruler
-    if (value) result[populationIdentifier] = value.valueString === 'true';
-    // If no value for a given population, we have an error
-    else {
+    if (value) {
+      if (valueType.valueString === 'Boolean') {
+        // NOTE: using === 'true' to get a boolean value from the returned string
+        result[populationIdentifier] = value.valueString === 'true';
+      } else if (valueType.valueString === 'List') {
+        // For a List return type in episode of care measures, we want a non-zero
+        // amount of episodes for this patient
+        const isEmptyPopulation = value.valueString === '[]';
+        const numEpisodes = isEmptyPopulation ? 0 : value.resource.entry.length;
+        result[populationIdentifier] = numEpisodes > 0;
+
+        // Add a column for number of episodes in the population for episode of care measures
+        if (!result.counts) {
```

Comment:
Thoughts on adding result.ids = <ids of episodes in the value.resource.entry list> To make it easier to debug _which_ episodes are or are not satisfying the logic.

Reply:
Yeah I can look at this. What do you think the best format is for outputting these into csv? Another column for `epsiodeIds` that contains a list of resource ids? Any other suggestions?

Reply:
I think a column that contains a list of the resource ids would be perfect

Reply:
Reviewing the column, it looks good on CMS104.